blog/_posts/2018-04-25-open-source-charts-2017.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: Kubernetes Community - Top of the Open Source Charts in 2017
+date: Wednesday, April 25, 2018
+---
+
+2017 was a huge year for Kubernetes, and GitHub’s latest [Octoverse report](https://octoverse.github.com) illustrates just how much attention this project has been getting.
+
+Kubernetes, an [open source platform for running application containers](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/), provides a consistent interface that enables developers and ops teams to automate the deployment, management, and scaling of a wide variety of applications on just about any infrastructure.
+
+Solving these shared challenges by leveraging a wide community of expertise and industrial experience, as Kubernetes does, helps engineers focus on building their own products at the top of the stack, rather than needlessly duplicating work that now exists as a standard part of the “cloud native” toolkit.
+
+However, achieving these gains via ad-hoc collective organizing is its own unique challenge, one which makes it increasingly difficult to support open source, community-driven efforts through periods of rapid growth.
+
+Read on to find out how the Kubernetes Community has addressed these scaling challenges to reach the top of the charts in GitHub’s 2017 Octoverse report.
+
+## Most-Discussed on GitHub
+
+The top two most-discussed repos of 2017 are both based on Kubernetes:
+
+![Most Discussed](/images/blog-logging/2018-04-24-open-source-charts-2017/most-discussed.png)
+
+Of all the open source repositories on GitHub, none received more issue comments than [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/). [OpenShift](http://openshift.com/), a [CNCF certified distribution of Kubernetes](https://blog.openshift.com/kubernetes-and-openshift-community-standards-and-certifications/
+https://www.cncf.io/announcement/2017/11/13/cloud-native-computing-foundation-launches-certified-kubernetes-program-32-conformant-distributions-platforms/), took second place.  
```

Comment:
The second link may not work

Reply:
@tengqm  I understand that this is for version management.